extensions/amp-form/amp-form.md

```diff
@@ -129,6 +129,30 @@ The value for `action-xhr` can be the same or a different endpoint than `action`
 
 To learn about redirecting the user after successfully submitting the form, see the [Redirecting after a submission](#redirecting-after-a-submission) section below.
 
+##### data-initialize-from-url
+
+Initializes supported form fields from the URL. When this attribute is present, `<input>`, `<select>`, and `<textarea>` fields under the form can optionally be initialized from URL query parameter values.
```

Comment:
Suggestion: "Initializes input fields from the document URL's search string, where the query parameter name matches the field's name."

Reply:
I'm ok with modifying this, but I'm not sure your suggestion is the right final text.

- This is the only place where the supported form field tags are written: `<input>`, `<select>`, and `<textarea>`.
- _From the document URL_ makes it sound like the URL of the AMP document matters more than `window.location` in a PWA. This is not the case.
- I've been trying to avoid using the word "input" when describing form fields because `<input>` is just one of three supported form fields (or form controls, which would be more accurate but less recognizable by readers).

Reply:
Ah, I just meant to supplement the first sentence. Window URL and "form field" sound fine.

Reply:
This should be resolved in 1894b76.